src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -2,28 +2,63 @@
  * @jest-environment jsdom
  */
 
-import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCoffee, faCircle } from '../__fixtures__/icons'
-import { compileAndMount, mountFromProps } from '../__fixtures__/helpers'
-import FontAwesomeIcon from '../FontAwesomeIcon'
+ import { library } from '@fortawesome/fontawesome-svg-core'
+ import { faCoffee, faCircle, faAlien } from '../__fixtures__/icons'
+ import { compileAndMount, mountFromProps } from '../__fixtures__/helpers'
+ import FontAwesomeIcon from '../FontAwesomeIcon'
 
-beforeEach(() => {
-  library.add(faCoffee, faCircle)
-})
+ beforeEach(() => {
+   library.add(faCoffee, faCircle, faAlien)
+ })
```

Comment:
@jasonlundien I think we need to have a clean up in here like this:

https://github.com/FortAwesome/react-fontawesome/blob/master/src/components/__tests__/FontAwesomeIcon.test.js#L13-L15